ingestion_program/systematics.py

```diff
@@ -610,6 +562,9 @@ def get_bootstraped_dataset(
 
     pseudo_data = []
     for key in test_set.keys():
+        temp = (test_set[key].sample(
```

Comment:
this line looks repeated

Reply:
yeah  it  is copied I screwed up a merge conflict 

please keep one the 2

RC